src/Todo.js

```diff
@@ -1,7 +1,39 @@
 'use strict';
 
 class Todo {
-  // write code here
+  constructor() {
+    this.list = [];
+    this.idCount = 1;
+  }
+  addItem(title, priority) {
+    this.list.push({
+      'title': title, 'priority': priority, 'id': this.idCount,
+    });
+    this.idCount++;
+
+    return this.idCount - 1;
+  };
+  removeItem(id) {
+    if (this.list.some(item => item.id === id)) {
+      this.list = this.list.filter(item => item.id !== id);
+
+      return true;
+    } else {
+      return false;
+    }
+  }
+  getItem(id) {
+    const findElem = this.list.find(item => item.id === id);
```

Comment:
Что такое elem?
findElem звучит как фунция
```suggestion
    const item = this.list.find(item => item.id === id);
```